tests/trainer/test_trainer_distributed.py

```diff
@@ -61,6 +62,22 @@ def forward(self, input_ids, labels=None):
             else:
                 return input_ids
 
+class TestTrainerDistributedXLA(TestCasePlus):
+    @require_torch_tpu
```

Comment:
We don't have any runner testing on XLA for now, so this test will never be run FYI.

Reply:
Thanks. Will get help with running these tests.